types/tampermonkey/index.d.ts

```diff
@@ -239,6 +224,123 @@ declare namespace Tampermonkey {
     }
 
     type NotificationDetails = TextNotification | HighlightNotification;
+
+    interface GM_info {
+        downloadMode: 'native' | 'browser' | 'disabled';
+        isFirstPartyIsolation: boolean | undefined;
+        isIncognito: boolean;
+
+        /**
+         * `.. | null` means if it was not explicitely set in the metadata
+         * block it is null
+         */
+        script: {
```

Comment:
Please create types nested objects

Reply:
Should I declare those in the tampermonkey namespace as well?

Reply:
Yes. I think every type should be exposed. Imagine a user method that accepts `script` object and it should be typed